collector/hyperv.go

```diff
@@ -593,6 +605,69 @@ func NewHyperVCollector() (Collector, error) {
 			[]string{"vm_interface"},
 			nil,
 		),
+
+		//
+
+		VMMemoryAddedMemory: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("vm_memory"), "added"),
+			"",
+			[]string{"vm"},
+			nil,
+		),
+		VMMemoryAveragePressure: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, buildSubsystemName("vm_memory"), "pressure_average"),
+			"This counter represents the average pressure in the VM.",
+			[]string{"vm"},
+			nil,
+		),
```

Comment:
For the `Gauge` metrics, it would be best to remove "counter" from the description to avoid confusion of the metric type.

Reply:
Sounds good. The descriptions started from the WMI property description. Fixed in the latest version.